### PR TITLE
feat: add infobox for elements at snapshotMaxDepth

### PR DESCRIPTION
**Pull request by eglitise**
This PR adds a new infobox when selecting an element located at `snapshotMaxDepth`. It helps users understand that some elements may be hidden, explains the reason why they are hidden, and provides a way to expose them.
image
The hope is that this infobox will also provide additional help for users testing React Native apps on iOS, since such apps [are known to have issues with high element depth](https://appium.github.io/appium-inspector/latest/troubleshooting/#cannot-see-full-source-tree-with-xcuitest-driver-react-native-app).

### CODE REVIEW

**Review by KazuCocoa — APPROVED** [review]
👍

File: app/common/public/locales/en/translation.json [hunk 1]
```diff
@@ -133,6 +133,7 @@
   "contextDropdownInfo": "Multiple contexts detected; certain elements might only be accessible after switching to a different context. Note that webview inspection in Appium Inspector is less accurate than the DevTools of Chrome or Safari. For more information, see:",
   "idAutocompletionCanBeDisabled": "The requested id selector does not have a package name prefix. This Appium session has package name autocompletion enabled, which may be the reason why no elements were found. To disable this behavior, relaunch this session with the capability 'appium:disableIdLocatorAutocompletion' set to 'true'.",
   "missingAutomationNameForStrategies": "Additional locator strategies may be available. To view them, add the capability 'appium:automationName' when launching the session. Note that this capability is mandatory in Appium 2.",
+  "snapshotMaxDepthReached": "This element is located at the maximum source depth {{selectedElementDepth}}, therefore its child elements (if any) are not shown. To reveal them, change the 'snapshotMaxDepth' setting to a value higher than {{selectedElementDepth}}. Check your driver documentation for the maximum supported value.",
```
Comment (mykola-mokhnach) [inline app/common/public/locales/en/translation.json:136]: should we also mention the maximum supported value is 62?

Comment (mykola-mokhnach) [reply app/common/public/locales/en/translation.json:136]: perhaps `greater than`?

Comment (eglitise) [reply app/common/public/locales/en/translation.json:136]: The maximum value of 62 refers only to the XCUITest driver, but for UiAutomator2 it seems to be 500.

Comment (mykola-mokhnach) [reply app/common/public/locales/en/translation.json]: I assume it would still be hard for users to find this value in docs themselves

Comment (mykola-mokhnach) [reply app/common/public/locales/en/translation.json]: for UIA2 the value does not really have an upper limit. I don't know exactly at which value the tree reader starts to freak out

**Review by mykola-mokhnach — APPROVED** [review]

**Pull request merged**